src/e2e_test/perf_tool/pkg/metrics/prometheus_recorder.go

```
@@ -43,10 +43,11 @@ import (
 )
 
 type prometheusRecorderImpl struct {
-	clusterCtx *cluster.Context
-	spec       *experimentpb.PrometheusScrapeSpec
-	eg         *errgroup.Group
-	resultCh   chan<- *ResultRow
+	clusterCtx    *cluster.Context
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(gci)